backend/helpers/helpers.go

```diff
@@ -53,6 +53,18 @@ func JsonWriter(w http.ResponseWriter, table interface{}) {
 	}
 }
 
+func JsonWriterString(w http.ResponseWriter, message string) {
+	messageStruct := struct {
```

Comment:
may be a way to make this one function just by type checking if it's a string, potentially "reflect.Typeof()"

Reply:
I just thought that would be weird since the parameter of the other one is called table

Reply:
i mean we can just rename it, I'm not particularly a fan of "JsonWriterString" being a separate function who's purpose is to remove quotes anyways

Reply:
Well it does a little more than that but maybe you can refactor it to your liking and cam can approve?

Reply:
that sounds good